pkg/flavor/kubernetes/nfs.go

```diff
@@ -763,6 +763,22 @@ func (flavor *Flavor) makeNFSDeployment(name string, nfsSpec *NFSSpec, nfsNamesp
 
 	volumes = append(volumes, configMapVol)
 
+        var seconds int64 = 30
```

Comment:
Would it be a huge effort to implement this as a `nfsTolerationSeconds` parameter in the `StorageClass`?

Reply:
As discussed, we will be taking this as part of another story because we need to change the code and test all over again which would cause a delay in handing over the build to QA for testing.